cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -342,6 +357,20 @@ def delete(self):
             {LOWEST: 'The selected Catalog Item was deleted',
              '5.11': 'The catalog item "{}" has been successfully deleted'.format(self.name)}))
 
+    def copy(self, name=None):
+        view = navigate_to(self, 'Copy')
+
+        if name:
+            view.name.fill(name)
+
+        view.add.click()
+        view.flash.assert_no_error()
+
+        # Catalog item can be any type
+        item_args = self.__dict__
+        item_args["name"] = view.name.value
+        return self.__class__(**item_args)
```

Comment:
Nice Implementation :100: :rocket:

Reply:
yup... I added this in state of `instantiation` because catalog item can be any `type` and instantiation will be hectic task in cases so I used this :dancer: